drug-interaction/cli/run.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
This file is actually generated using the [generate_scripts.py](https://github.com/linqs/psl-examples/blob/main/.templates/generate_scripts.py) script and [config options](https://github.com/linqs/psl-examples/blob/main/.templates/config.json).
This is so all examples get the same versions.